src/WebJobs.Script/Config/ScriptApplicationHostOptions.cs

```diff
@@ -31,5 +31,11 @@ public class ScriptApplicationHostOptions
         public IServiceProvider RootServiceProvider { get; set; }
 
         public bool IsStandbyConfiguration { get; internal set; }
+
+        public bool IsFileSystemReadOnly { get; set; }
+
+        public bool IsZipDeployment { get; set; }
```

Comment:
`IsZipDeployment` is only used in `ScriptApplicationHostOptionsSetup` - no need to expose it here. Calculate directly in `ScriptApplicationHostOptionsSetup`

Reply:
Thanks, good catch. Due to the refactoring, `IsZipDeployment` doesn't need to be exposed anymore. Updated!